rerun_cpp/src/rerun/components/class_id.cpp

```diff
@@ -1,4 +1,4 @@
-// DO NOT EDIT!: This file was auto-generated by crates/re_types_builder/src/codegen/cpp/mod.rs:54.
```

Comment:
offtopic: The line number in here is really silly an causes lots of unnecessary diff

Reply:
fixed